manifests/config.pp

```diff
@@ -16,6 +16,7 @@
 
   concat::fragment { 'base':
     target  => 'pulpcore settings',
+#    content => template('pulpcore/settings.py.erb', 'pulpcore/settings-mongodb.py.erb'),
```

Comment:
Not a fan of commented code :)

Reply:
Just a temporary test. I can't run them locally due to some environmental issues that I'm putting off resolving until after this is complete.